resource.go

```diff
@@ -5,7 +5,7 @@ package swid
 
 // Resource models a resource-entry
 type Resource struct {
+	ResourceExtension
```

Comment:
Minor comment: Any reason this appears first? As i understand, it is always Type, Global Attributes and then Extensions?

Reply:
I had to move it because the linter was moaning about the packing of the structure not being optimal.